src/BuildScriptGenerator/DefaultBuildScriptGenerator.cs

```diff
@@ -241,8 +241,8 @@ private static string GetBenvArgs(IDictionary<string, string> benvArgsMap)
         private IList<BuildScriptSnippet> GetBuildSnippets(
             BuildScriptGeneratorContext context,
             Dictionary<string, string> toolsToVersion,
-            List<string> directoriesToExcludeFromCopyToIntermediateDir,
-            List<string> directoriesToExlcudeFromCopyToBuildOutputDir)
+            [CanBeNull] List<string> directoriesToExcludeFromCopyToIntermediateDir,
```

Comment:
Please revert all these type of unrelated changes!

Reply:
They aren't unrelated - i changed the way this function works, so that it can accept `null`s in these two parameters.
The new method `GetRequiredToolVersions` relies on this change.